e2e/viewer_content_services_component.e2e.ts

```diff
@@ -109,116 +100,127 @@ describe('Content Services Viewer', () => {
         viewerPage.checkScalePageButtonIsDisplayed();
     });
 
-    it('2. Use viewer pagination', () => {
+    it('[C260040] Check pagination for a .pdf file', () => {
         viewerPage.clickNextPageButton();
         viewerPage.checkFileContent('2', pdfFile.secondPageText);
         viewerPage.checkPageSelectorInputIsDisplayed('2');
+
         viewerPage.clickPreviousPageButton();
         viewerPage.checkFileContent('1', pdfFile.firstPageText);
         viewerPage.checkPageSelectorInputIsDisplayed('1');
-        viewerPage.enterPage(pdfFile.lastPageNumber);
-        viewerPage.checkFileContent(pdfFile.lastPageNumber, pdfFile.lastPageText);
-        viewerPage.checkPageSelectorInputIsDisplayed(pdfFile.lastPageNumber);
 
-        viewerPage.canvasHeight().then(function (value) {
-            defaultHeight = parseInt(value, 10);
-        });
-        viewerPage.canvasWidth().then(function (value) {
-            defaultWidth = parseInt(value, 10);
-        });
+        viewerPage.clearPageNumber();
+        viewerPage.checkPageSelectorInputIsDisplayed('');
+
+        zoom = viewerPage.getZoom();
         viewerPage.clickZoomInButton();
-        viewerPage.canvasHeight().then(function (value) {
-            zoomedInHeight = parseInt(value, 10);
-            expect(zoomedInHeight).toBeGreaterThan(defaultHeight);
-        });
-        viewerPage.canvasWidth().then(function (value) {
-            zoomedInWidth = parseInt(value, 10);
-            expect(zoomedInWidth).toBeGreaterThan(defaultWidth);
-        });
-        viewerPage.clickScalePageButton();
-        viewerPage.canvasHeight().then(function (value) {
-            scaledHeight = parseInt(value, 10);
-            expect(scaledHeight).toEqual(defaultHeight);
-        });
-        viewerPage.canvasWidth().then(function (value) {
-            scaledWidth = parseInt(value, 10);
-            expect(scaledWidth).toEqual(defaultWidth);
-        });
+        viewerPage.checkZoomedIn(zoom);
+
+        zoom = viewerPage.getZoom();
         viewerPage.clickZoomOutButton();
-        viewerPage.canvasHeight().then(function (value) {
-            zoomedOutHeight = parseInt(value, 10);
-            expect(zoomedOutHeight).toBeLessThan(defaultHeight);
-        });
-        viewerPage.canvasWidth().then(function (value) {
-            zoomedOutWidth = parseInt(value, 10);
-            expect(zoomedOutWidth).toBeLessThan(defaultWidth);
-        });
-    });
+        viewerPage.checkZoomedOut(zoom);
 
-    it('3. Use viewer toolbar', () => {
-        viewerPage.clickDownloadButton();
-        viewerPage.clickInfoButton();
-        viewerPage.checkInfoSideBarIsDisplayed();
-        viewerPage.clickInfoButton();
-        viewerPage.checkInfoSideBarIsNotDisplayed();
-        expect(Util.fileExists(downloadedPdfFile, 10)).toBe(true);
         viewerPage.clickCloseButton();
     });
 
-    it('4. Open viewer for a .jpg file', () => {
+    it('3. C260042 : Check viewer toolbar', () => {
```

Comment:
the name of the test other test have to follow the naming convetion [number] description